std/math.d

```diff
@@ -2574,6 +2574,9 @@ unittest
              tuple(-T.infinity, -T.infinity, int.min),
              tuple(T.nan, T.nan, int.min),
              tuple(-T.nan, -T.nan, int.min),
+
+             // Phobos issue #16026:
+             tuple(3 * (T.min_normal * T.epsilon), T( .75), (T.min_exp - T.mant_dig) + 2)
```

Comment:
Well, it's just a comment, so obviously it doesn't **need** to be there. This is a common practice in the core **dlang** projects though: to label tests with the issue number that precipitated their inclusion.

Two main benefits I see:

* Seeing issue labels like this scattered throughout the tests provides real-world evidence to people browsing the code about what kind of test cases are needed for full coverage of related algorithms. Without that feedback, it is hard to know when to stop adding test cases.

* For anyone tempted to reduce the number of test cases while refactoring, or while trying to speed up the test suite, it says, "*Don't remove this one!*"